docs/workflow.rst

```diff
@@ -0,0 +1,231 @@
+********
+Workflow
+********
+
+This page provides a brief explanation of the workflow to perform APR calculations with `pAPRika`. For more detail users
+are recommended to go through the tutorials, which details further on how to setup and run APR simulations from start to
+finish.
+
+.. figure :: _static/images/flowchart.png
+   :figwidth: 500px
+   :width: 170px
+   :align: center
+
+   Flowchart of the *pAPRika* workflow for a typical APR simulation.
+
+Structure Preparation
+---------------------
+The starting structure for the APR simulation can be configured with *pAPRika*. The ``Align`` module provides functions
+to shift and orient a structure. For example, we can translate a structure to the origin and then orient the system to
+the :math:`z`-axis by running
+
+.. code ::
+
+    from paprika.align import *
+
+    translated_structure = translate_to_origin(structure)
+    aligned_structure = zalign(translated_structure, ":GST@C1", ":GST@C2")
+
+We use dummy atoms to define the a reaction coordinate for to pull the guest molecule. We can add dummy atoms to a
+structure using the ``Dummy Atoms`` module in *pAPRika*.
+
+.. code ::
+
+    from paprika.dummy import add_dummy
+
+    structure = dummy.add_dummy(structure, residue_name="DM1", z=-6.0)
+    structure = dummy.add_dummy(structure, residue_name="DM2", z=-9.0)
+    structure = dummy.add_dummy(structure, residue_name="DM3", z=-11.2, y=2.2)
+    structure.save("aligned_with_dummy.pdb", overwrite=True)
+
+We can use the ``tleap`` wrapper to combine all of these components to generate `AMBER` topology and coordinate files.
```

Comment:
Do you need a line to generate the dummy MOL2 files here?

Reply:
good point added.